_posts/2015-11-18-Stats-2015.markdown

```diff
@@ -0,0 +1,123 @@
+So, to wrap this up, I was looking at the Google Analytics for the CocoaPods web properties. My favourite odd statistic is this: We get more user sessions from Windows devices than we do iOS. Perhaps this mobile thing is just a fad?
+
+## CocoaPods
+
+So this year was a crazy year from the perspective of Apple releases. 
+
+We got to a point where in the last 3 months, Swift accounts for half of the new Pods. At this point last year we were just starting to ship betas supporting frameworks, enabling Swift for those brave enough to be early adopters. Now there's a Swift library is in the top ten downloaded Pods and there are tens of thousands of apps using CocoaPods with Swift frameworks.
+
+CocoaPods Rome?
+
+Apple also kept us all on our toes by introducing 2 new platforms. watchOS first came as an iOS extension, but then migrated to a real platform of its own with watchOS2. tvOS came towards the end of the year. This means CocoaPods now supports 4 independent platforms.
+
+We added determinate UUID generation, which is only important if you check in your `Pods` directory. This means running `pod install` would generate the same UUIDs for the files it generates, and you won't see diffs for `Pods.xcodeproj`.
+
+The CocoaPods plugin API got more powerful via `pre-install` and `post-install` hooks. Third party adoption of plugins has been great, and we've moved a bunch of non-core CocoaPods code into plugins. For example; trunk, stats, search and try are all plugins.
+
+We made it harder to accidentally edit your Pods files by having the files read-only by default.
+
+Finally, the introduction of Molinillo made CocoaPods' dependency resolving first class. This means that a `pod install` is now always going to be the exact same Pods as the one that generated the `Podfile.lock`. Also, seriously, it's in RubyGems + Bundler, that's so cool.
+
+## Stats
+
+Alright, so, let's check out on some interesting numbers:
+
+- 24 searches per minute.
+- 45 pod installs per minute.
+- 100,000 unique app targets.
+- 100,000 uniques to the site.
+- 500,000 monthly page views.
+- 2.5 million downloads.
+- 6.6 million targets integrated by `pod install` / `pod update`.
+- 30 million pods have been integrated by CocoaPods.
+
+The Cocoa community is definitely growing!
+
+## CocoaPods.app
+
+The final major project that we've been working on, and have shipped a few releases sneakily is CocoaPods.app. In our constant work to lower the barrier to entry for building apps, we've started tackling the last mile. How do we improve, and simplify the installation and updating of CocoaPods?
+
+The best way to do that is to make CocoaPods act like other tools a Cocoa developer would work with every day. So we have been working on a CocoaPods.app that you can drag into `/Applications` and it will deal with handling CocoaPods for you.
+
+Sorry if using the terminal just for this made you feel like a leet hacker, it's still an option with this though, just not the primary use case.
```

Comment:
This is not true imo. The initial primary use case was terminal use, now I’d say it’s equal. The point of the app is to be a convenient package regardless of how you’d like to use CP.